atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -147,10 +163,26 @@ def get_buildroot(self, worker_metadatas):
         :return: list, containing dicts of partial metadata
         """
         buildroots = []
-        for platform in sorted(worker_metadatas.keys()):
-            for instance in worker_metadatas[platform]['buildroots']:
-                instance['id'] = '{}-{}'.format(platform, instance['id'])
-                buildroots.append(instance)
+
+        if self.source_build:
```

Comment:
I see this anti-pattern here, almost every function starts with `if self.source_build`. Can we create 2 classes with inheritance and set in inner.json which should be called?

Reply:
do you mean those methods ? is_image_build, is_source_container_build

Reply:
no, we discussed offline, resolving this